packages/dapp/src/const/MenuConfig/useMainMenuContent.tsx

```diff
@@ -28,6 +31,20 @@ export const useMainMenuContent = () => {
   const themeMode = useSettingsStore((state) => state.themeMode);
   const onOpenSupportModal = useMenuStore((state) => state.onOpenSupportModal);
 
+  const utmMinMenuJson = {
+    utm_medium: 'menu',
+  };
+
+  const explorerUrl = appendUTMParametersToLink('https://explorer.li.fi/', {
+    ...utmMinMenuJson,
+    utm_campaign: 'jumper_to_explorer',
+  });
+
+  const lifiUrl = appendUTMParametersToLink('https://li.fi/', {
+    ...utmMinMenuJson,
+    utm_campaign: 'jumper_to_lifi',
+  });
+
```

Comment:
See comment above 🙂

Reply:
I wanted to be really smart here and work out DRYest code! :D 

I have used that approach as well on lifi-website. Guessing, that there might be cases where this could be useful. But overall, type definitions for repetitive labels might also do the trick just fine. Thanks for the hint!